include/depthimage_to_laserscan/DepthImageToLaserScan.h

```diff
@@ -27,24 +27,27 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
-/* 
+/*
```

Comment:
Copyright format should be updated to pass copyright linter test, same for every file in this package